packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-dynamic-intervention-policy-group.vue

```diff
@@ -26,7 +26,7 @@ const props = defineProps<{
 
 const emit = defineEmits(['update-self']);
 
-const dynamicInterventions = ref<DynamicIntervention[]>(props.config.intervention.dynamicInterventions);
+const dynamicInterventions = computed<DynamicIntervention[]>(() => props.config.intervention.dynamicInterventions);
```

Comment:
The static intervention are not declared as computed, what makes dynamic special that a computed wrapper is needed?

```
... in tera-static-intervention-policy.vue...
const staticInterventions = ref<StaticIntervention[]>(props.config.intervention.staticInterventions);
```

Reply:
Static interventions are editable within optimize so I wouldn't be able to just switch to a computed value whereas dynamic interventions currently are not editable with optimize.   I was mostly just avoiding trying to refactor the component by using the watcher in the static interventions component 🙈.